packages/components/.storybook/main.js

```diff
@@ -1,9 +1,5 @@
 module.exports = {
-  stories: [
-    "**/*.stories.mdx",
-    "../src/**/*.stories.@(ts|tsx|mdx)",
-    "**/*.stories.@(ts|tsx|mdx)",
-  ],
+  stories: ["../src", "../docs"],
```

Comment:
Couldn't find a way to get Storybook 6.4 to look at stories in its own .storybook/ directory. So I moved the stories there into the docs/ directory.

Not saying it's not possible, but I couldn't find an obvious way to do it.

Reply:
`docs/` kinda makes more sense anyways 👍